.github/workflows/helm.yml

```diff
@@ -39,3 +39,13 @@ jobs:
           config: helm/cr.yaml
           charts_dir: helm/
           mark_as_latest: false
+
+  invoke_pages_build:
+    runs-on: ubuntu-latest
```

Comment:
I think we need a `needs: [ release ]`, as otherwise this might run concurrently with the release.